src/components/popover/examples/popover-trigger-interaction.tsx

```diff
@@ -0,0 +1,124 @@
+ * Provide clear visual indicators, or use accessible attributes like `title` or `aria-label`,
+ * to signal that the trigger element is interactive when hovered or focused.
+ * This ensures that users understand they can interact with the element even without clicking.
+ *
+ * **Low discoverability**:
+ * Remember that such interaction are not easily discovered by the end-users.
+ * Therefore, you should avoid having any critical functionality to be hidden
+ * behind such interactions, and only use it as supplementary means of enriching
+ * the user experience.
+ * :::
+ */
+
+@Component({
+    tag: 'limel-example-popover-trigger-interaction',
+    shadow: true,
+})
+export class PopoverTriggerInteractionExample {
+    @State()
+    private isOpen = false;
+
+    private hoverTimeout: number;
+
+    public render() {
+        const chipImage = {
+            src: 'https://lundalogik.github.io/lime-elements/0e6f74c0-11d9-465b-aac6-44f33da3cb7c.png',
+            alt: 'A picture of Adrian Schmidt, Head of Smooth Operations at Lime Technologies',
+        };
+
+        const markdown = `
+![Adrian Schmidt](https://lundalogik.github.io/lime-elements/0e6f74c0-11d9-465b-aac6-44f33da3cb7c.png)
+### Adrian Schmidt
+Head of
+Smooth Operations
+
+[Get in touch](https://github.com/adrianschmidt)
+        `;
+
+        return [
+            <limel-popover
+                open={this.isOpen}
+                onClose={this.onPopoverClose}
+                open-direction="top"
+            >
+                <limel-chip
+                    text="Adrian Schmidt"
+                    image={chipImage}
+                    slot="trigger"
+                    title="Hover or focus me!"
+                    onMouseEnter={this.openPopoverWithDelay}
+                    onFocus={this.openPopoverWithDelay}
+                    onClick={this.handleClick}
+                />
+                <limel-markdown
+                    style={{
+                        display: 'flex',
+                        'flex-direction': 'column',
+                        gap: '0.5rem',
+                        padding: '0.5rem',
+                    }}
+                    tabIndex={0}
+                    value={markdown}
+                />
+            </limel-popover>,
+        ];
+    }
+
+    private openPopoverWithDelay = (event: FocusEvent | MouseEvent) => {
+        event.stopPropagation();
+        console.log('opening popover with delay');
+
+        clearTimeout(this.hoverTimeout);
+
+        this.hoverTimeout = window.setTimeout(() => {
+            this.isOpen = true;
+        }, TRIGGER_DELAY);
+    };
+
+    private onPopoverClose = (event: CustomEvent) => {
+        event.stopPropagation();
+        console.log('closing popover instantly');
+        this.isOpen = false;
+    };
+
+    private handleClick = (event: MouseEvent) => {
+        event.stopPropagation();
+        console.log('chip is clicked, closing popover and opening alert');
+        this.isOpen = false;
```

Comment:
we have an old untreated issue here https://github.com/Lundalogik/lime-elements/pull/1197 

Basically, while the portal is open, it prevents clicks on other clickable elements in the UI. This becomes clear here in this example. I'm not sure if we should talk about it in the docs, as I consider it as a bug. But just mentioning it here, since I don't want it as a comment 😅 Also maybe you who are reviewing has some magical idea about the correct implementation for such things, so that we can prevent these buggy behaviors